docs/book/src/topics/managedcluster.md

```diff
@@ -197,6 +197,31 @@ spec:
 ---
 ```
 
+### AKS AAD
+
+Azure Kubernetes Service can be configured to use Azure Active Directory for user authentication.
```

Comment:
can you please add a link to the AKS AAD documentation here? https://docs.microsoft.com/en-us/azure/aks/managed-aad

Reply:
sure 👍